src/status_im2/common/bottom_sheet_screen/view.cljs

```diff
@@ -0,0 +1,75 @@
+(ns status-im2.common.bottom-sheet-screen.view
```

Comment:
this screen doesn't look like common, it looks like a special screen, other similar screen will be more general and should be simpler

Reply:
https://github.com/status-im/status-mobile/pull/15399#issuecomment-1473600268